src/theme/components/Terminal/rpc/GetBlock.js

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from "react";
```

Comment:
Is there any way to control the speed of the text showing up here? It's kinda slow for longer commands

Reply:
It looks like we could modify the `Terminal` component slightly to take in an option `typeDelay` and pass that on to `Termynal`